src/models/UIOptions.ts

```diff
@@ -0,0 +1,11 @@
+export type UIOptions = {
+  orbitSpeed: number
+  showGrid: number
+  isUIOptionsPopupVisible: true | false
```

Comment:
I'd probably just use the `boolean` type here.

Reply:
I have submitted changes to honor your suggestions!